v2/rscg_examples_site/docs/indexRSCG.md

```
@@ -20,6 +20,7 @@ This is the list of 251 ( 16 from Microsoft) RSCG with examples
 
 | No        | Name  | Date | Category |
 | --------- | ----- | ---- | -------- |  
+|252| [RSCG_idempotency by Ignat Andrei ](/docs/RSCG_idempotency)|2026-01-28 => 28 January 2026 | [Idempotency](/docs/Categories/Idempotency) |
```

markdownlint-cli2 (0.18.1)

Spaces inside link text

(MD039, no-space-in-links)